packages/protocol/thread/thread.ts

```diff
@@ -662,7 +659,7 @@ class _ThreadFront {
   }
 
   async loadAsyncParentFrames() {
-    await this.ensureAllSources();
+    await this.getCurrentPause().isLoaded();
```

Comment:
I will say, if we're going to call `get/ensureCurrentPause` I much prefer getting this Promise back. Otherwise, we call it but it's not async and it doesn't actually ensure anything (which I'm guessing is why the name changed 😉)